src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx

```diff
@@ -12,21 +12,34 @@ If your team deploys software and cloud services through Azure, our [Azure Nativ
 * Deploy New Relic APM and infrastructure agents through an automated setup flow 
 * Manage the metrics and logs for all your resources through an interactive dashboard
 
-## What you can monitor [#features]
+## Features [#features]
 
-You can deploy the following agents through the Azure Native New Relic service:
+The features and integrations available for the Azure Native New Relic Service is different from regular, non-Azure New Relic. 
 
-APM agents:
+You can use the following New Relic agents:
+
+**APM agents:**
 
 * .NET
 * Java
 
-Infrastructure agents: 
+**Infrastructure agent for:** 
 
 * Windows
 * Linux 
 
-[Learn more about benefits on the Azure Marketplace.](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg)
+### Unavailable features [#data-plus]
+
+Our Standard and Pro [editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are available, but not yet Enterprise.
+
+This service comes with our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus), but currently these Data Plus features are **not** yet available: 
```

Comment:
This service comes with our [Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus). Not all Data Plus features are available to Azure Native New Relic Service subscribers, please see the Data Plus [documentation](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing/#data-plus).